Dockerfile

```diff
@@ -22,4 +22,5 @@ ARG SWAGGER
 
 COPY . /src
 WORKDIR /src
+RUN git checkout .
```

Comment:
What is this for?

Reply:
we will check the repo status when to build.  such as: `v1.0.0-alpha.1.18+d9be6353138498-dirty`, if the repo was changed, we will add a `dirty` field to the message. So we add `git checkout .` to make sure the repo is clean.